integration-tests/zombienet/src/lib.rs

```diff
@@ -28,11 +27,13 @@ pub fn small_network() -> Result<NetworkConfig, Error> {
 				.with_node(|node| node.with_name(BOB))
 		})
 		.with_parachain(|p| {
-			p.with_id(2000).cumulus_based(true).with_collator(|n| {
-				n.with_name(COLLATOR)
-					.with_command("polkadot-parachain")
-					.with_image(images.cumulus.as_str())
-			})
+			p.with_id(1005)
```

Comment:
intentional change from 2000 to 1005?

Reply:
yes, I change some default cumulus parachain to coretime-polkadot-local